common/static/sass/edx-pattern-library-shims/_buttons.scss

```diff
@@ -0,0 +1,121 @@
+// ------------------------------
+// LMS Problem Feedback Revamp styling
+// Mirror styles from the Pattern Library
+
+// NOTE: This is the left-to-right (LTR) configured style compile.
+// It should mirror lms-problem-feedback-rtl w/ the exception of bi-app references.
+
+@import 'base/variables';
+
+
+// ----------------------------
+// #GLOBALS
+// ----------------------------
+%btn {
+  display: inline-block;
+  border-style: $btn-border-style;
+  border-radius: $btn-border-radius;
+  border-width: $btn-border-size;
+  box-shadow: none;
+  padding: 0.625rem 1.25rem;
+  font-size: 16px;
+  font-weight: normal;
+  text-shadow: none;
+  text-transform: capitalize;
+
+  // Display: block, one button per line, full width
+  &.block {
+    display: block;
+    width: 100%;
+  }
+
+  // STATE: is disabled
+  &:disabled,
+  &.is-disabled {
+    @extend %state-disabled;
+  }
+
+  .icon {
+    display: inline-block;
+    vertical-align: baseline;
+
+    &:only-child,
+    .sr-only + & {
+      margin-right: 0;
```

Comment:
This should be as follows to be bi-directional:

    @include margin-right(0);